pkg/registries/registries.go

```diff
@@ -20,6 +20,15 @@ func scopeMatchesRegistry(scope, reg string) bool {
 	return false
```

Comment:
Above, AFAICS `scopeMatchesRegistry("*.foo/bar", "*.foo")`. That doesn’t really matter because the former scope is invalid, but integrating the two functions could avoid this.

Reply:
Moved into one function.